### PR TITLE
Run checks in a container

### PR DESCRIPTION
**Pull request by poncovka**
* Add support for running checks and tests in a container.
* Call `make container-test` to run tests in a container.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+FROM fedora:rawhide
+
+RUN dnf update -y \
```
Comment (VladimirSlavik) [inline Dockerfile:3]: Not sure if it matters in this case, but usually people do `set -eu` or such first thing to help with debugging.

**Review by VladimirSlavik — APPROVED** [review]
Looks good to me. Thank you!

File: Makefile [hunk 2]
```diff
@@ -24,10 +25,16 @@ _default:
 	@echo "Put hello_world_addon_updates.img up where you can use it via"
 	@echo "  inst.updates=<path>/hello_world_addon_updates.img"
 
+.PHONY: container-check
+container-test:
```
[the remark this answers is not in the document]
Comment (poncovka) [reply Makefile]: Fixed, thanks!

File: Dockerfile (hunk 1 above)
Comment (poncovka) [reply Dockerfile:3]: * `-e` shouldn't be necessary, since I use everywhere `&&`
* `-u` changes the behaviour of unset variables, I don't use any

**Comment by poncovka** [comment]
Updated.

**Review by jkonecny12 — APPROVED** [review]
LGTM!

**Pull request merged**